src/theme.js

```diff
@@ -0,0 +1,20 @@
+export const theme = {
+  color: {
+    woodsmoke: "#18181B",
+    white: "#FFFFFF",
+    waterloo: "#7E839A",
+    mercury: "#E5E5E5",
+    mystic: "#E4E6F0",
+    candlelight: "#FCD420",
+    pattensblue: "#D6E4FF",
+    scienceblue: "#0044CC",
+    silver: "#C4C4C4",
+    snuff: "#DDDDEE",
+    black: "#000000",
+  },
+  breakpoint: {
+    mobileMax: "767px",
+    tabeltMax: "992px",
```

Comment:
Mała literówka się wkradła "tabeltMax", a tak to według mnie dobra robota! nie mam żadnych zastrzeżeń :)

Reply:
Super! Fajnie, że już zaczęło się coś dziać 🥇 
Tak jak napisałem możemy jedynie darować sobie importowanie reacta w każdym index.js ponieważ od nowszej wersji, nie trzeba tego robić 👍